src/Bicep.Core/LanguageConstants.cs

```diff
@@ -119,10 +119,16 @@ public static class LanguageConstants
 
         public const string McrRepositoryPrefix = "bicep/";
 
-        // https://github.com/opencontainers/image-spec/blob/main/annotations.md
+        // From https://github.com/opencontainers/image-spec/blob/main/annotations.md
+        //
+        // URL to get documentation on the image (string)
         public const string OciOpenContainerImageDocumentationAnnotation = "org.opencontainers.image.documentation";
+        // Human-readable description of the software packaged in the image (string)
         public const string OciOpenContainerImageDescriptionAnnotation = "org.opencontainers.image.description";
+        // date and time on which the image was built, conforming to RFC 3339.
         public const string OciOpenContainerImageCreatedAnnotation = "org.opencontainers.image.created";
+        // Human-readable title of the image (string)
+        public const string OciOpenContainerImageTitleAnnotation = "org.opencontainers.image.title";
```

Comment:
I think the LanguageConstant class is getting a bit messy. Would you mind creating a new constant class under `https://github.com/Azure/bicep/tree/main/src/Bicep.Core/Registry/Oci` and move the Bicep registry constants to the new class?

Reply:
Makes sense. 997d212d2